### ISSUE TITLE
Timezone changes for Jordan/Amman not updated

### ISSUE THREAD

**Issue by emahama** [moment/moment-timezone#1015]
### Moment-timezone version which you use: 0.5.38
Version:

_Note: many issues are resolved if you just upgrade to the latest version_

### Issue description:
On October 5, 2022, the Jordanian government made an official announcement ending the winter-time Daylight Saving Time (DST) time zone change. Starting at 12:00 a.m. Friday, October 28, 2022, the official time will not advance by an hour and will permanently shift to the UTC + 3 time zone.

The impact of this change is as follows:

Clocks will not be advanced by an hour at 12:00 a.m. on October 28, 2022 for the Jordan time zone.
The Jordan time zone will permanently shift to the UTC + 3 time zone. 

See link for your reference: 
https://techcommunity.microsoft.com/t5/daylight-saving-time-time-zone/interim-guidance-for-jordan-end-of-winter-time-dst-2022/ba-p/3658533#:~:text=On%20October%205%2C%202022%2C%20the%20Jordanian%20government%20made,The%20impact%20of%20this%20change%20is%20as%20follows%3A

It appears this is not reflected in moment-timezone. I am currently on the latest version, but I see UTC + 2 instead of UTC +3

**Comment by gilmoreorless** [comment]
The change for Jordan was released in the IANA time zone database's `2022e`, which was included in Moment Timezone 0.5.38 (#1009). When I run it locally I see the correct date:

```js
moment.tz.version;
// '0.5.38'
moment.tz('Asia/Amman').format();
// '2022-11-12T14:12:33+03:00'
```

Perhaps there's something in your setup or environment that's pulling in stale data?

**Comment by emahama** [comment]
Thank you for your prompt response. I will check my environment.
